### PR TITLE
283-fix: fix 'https://rollingscopes.com/' redirect

### LINKED ISSUE #283: Fix redirect from 'https://rollingscopes.com/' to 'https://rs.school/'
## 🐛 Describe the Bug

When navigating to 'https://rollingscopes.com/', the expected redirect to 'https://rs.school/' does not occur. Instead, the user is kept on the initial page (or there might be other incorrect behavior). This bug prevents the user from being led to the correct site automatically.

## 🔄 Steps to Reproduce
1. Navigate to 'https://rollingscopes.com/'
2. Click on 'Community' in header
3. Observe the error

## 🏁 Expected Behavior
When a user navigates to 'https://rollingscopes.com/', they should be automatically redirected to 'https://rs.school/'. This expected behavior ensures the user lands on the correct site.

## 📸 Screenshots

Image

## ✅ Definition of Done (DoD):

The website 'https://rollingscopes.com/' successfully redirects to 'https://rs.school/' in all cases.

### PR DESCRIPTION
**Pull request by natanchik**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description

1. Fixed redirect 'https://rollingscopes.com/' to 'https://rs.school/'. To check if the redirect works:
- follow the link 'https://pr332.rollingscopes.com/community';
- an automatic redirect to 'https://rs.school/' will take place.

2. Updated the names of web pages in the browser tab

3. Removed `#main` from URLs.

Changes 2, 3 can be checked on the 'https://pr332.rs.school/' link.

## Screenshots

Before
Screenshot 2024-06-24 195945

Now
image

Before
Screenshot 2024-06-24 200117

Now
Screenshot 2024-06-24 200235

Screenshot 2024-06-24 200248

Before
Screenshot 2024-06-24 201007

Now
image

## Related Tickets & Documents

- Related Issue #283
- Closes #283

## Added/updated tests?

- [x] 👌 Yes
- [ ] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

File: src/pages/community.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import { Events } from '@/widgets/events';
+import { Hero } from '@/widgets/hero';
+import { Merch } from '@/widgets/merch';
+import { Numbers } from '@/widgets/numbers';
+import { Pictures } from '@/widgets/pictures';
+import { Places } from '@/widgets/places';
+import { Speakers } from '@/widgets/speakers';
+import { Support } from '@/widgets/support';
+
+export const CommunityPage: FC = () => {
```
Comment (NikitaStarmoussov) [inline src/pages/community.tsx:16]: name without Page

File: src/app/routes.tsx [hunk 2]
```diff
@@ -98,56 +97,33 @@ const notFoundRoute: RouteObject = {
   },
 };
 
-export const routes: RouteObject[] = config.isRollingScopesLanding
-  ? [
+export const routes: RouteObject[] = [
+  {
+    path: ROUTES.HOME,
+    lazy: async () => {
+      const { BaseLayout } = await import('./layouts/base-layout');
```
Comment (NikitaStarmoussov) [inline src/app/routes.tsx:104]: use absolute path @

File: src/pages/community.tsx (hunk 1 above)
Comment (natanchik) [reply src/pages/community.tsx]: changed

File: src/app/routes.tsx (hunk 2 above)
Comment (natanchik) [reply src/app/routes.tsx]: done

File: src/widgets/breadcrumbs/ui/breadcrumbs.tsx [hunk 3]
```diff
@@ -17,14 +16,14 @@ export const Breadcrumbs = () => {
       <div className="breadcrumbs content">
         <ul>
           <li>
-            <Link to={buildUrl('/')} className="link">
+            <Link to="/" className="link">
```
Comment (Quiddlee) [inline src/widgets/breadcrumbs/ui/breadcrumbs.tsx:19]:
```suggestion
            <Link to={ROUTES.HOME} className="link">
```

File: src/widgets/mobile-view/ui/mobile-view.tsx [hunk 4]
```diff
@@ -19,11 +18,11 @@ export const MobileView = ({ type }: MobileViewProps) => {
 
   return (
     <div className="mobile-view" data-testid="mobile-view">
-      <Link to={buildUrl('/')}>
+      <Link to="/">
```
Comment (Quiddlee) [inline src/widgets/mobile-view/ui/mobile-view.tsx:21]:
```suggestion
      <Link to={ROUTES.HOME}>
```

File: src/widgets/mobile-view/ui/mobile-view.tsx [hunk 5]
```diff
@@ -19,11 +18,11 @@ export const MobileView = ({ type }: MobileViewProps) => {
...
         <LogoWrapper type={type} />
       </Link>
 
-      <Link to={buildUrl('/#about')} className={`main-link ${color}`}>
+      <Link to="/community/#about" className={`main-link ${color}`}>
```
Comment (Quiddlee) [inline src/widgets/mobile-view/ui/mobile-view.tsx:25]:
```suggestion
      <Link to=`/${ROUTES.COMMUNITY}/#about` className={classNames('main-link', color)}>
```

File: src/widgets/mobile-view/ui/mobile-view.tsx [hunk 6]
```diff
@@ -37,19 +36,19 @@ export const MobileView = ({ type }: MobileViewProps) => {
 
       <Divider type={type} />
 
-      <Link to={buildUrl('/#community')} className={`main-link mt ${color}`}>
+      <Link to="/community/#community" className={`main-link mt ${color}`}>
```
Comment (Quiddlee) [inline src/widgets/mobile-view/ui/mobile-view.tsx:39]:
```suggestion
      <Link to=`/${ROUTES.COMMUNITY}/#community` className={classNames('main-link', 'mt', color)}>
```

File: src/widgets/mobile-view/ui/mobile-view.tsx [hunk 7]
```diff
@@ -37,19 +36,19 @@ export const MobileView = ({ type }: MobileViewProps) => {
...
         Community
       </Link>
 
       <Divider type={type} />
 
-      <Link to={buildUrl('/#events')} className={`main-link ${color}`}>
+      <Link to="/community/#events" className={`main-link ${color}`}>
```
Comment (Quiddlee) [inline src/widgets/mobile-view/ui/mobile-view.tsx:45]:
```suggestion
      <Link to=`/${ROUTES.COMMUNITY}/#events` className={classNames('main-link', color)}>
```

File: src/widgets/mobile-view/ui/mobile-view.tsx [hunk 8]
```diff
@@ -37,19 +36,19 @@ export const MobileView = ({ type }: MobileViewProps) => {
...
         Events
       </Link>
 
       <Divider type={type} />
 
-      <Link to={buildUrl('/#merch')} className={`main-link ${color}`}>
+      <Link to="/community/#merch" className={`main-link ${color}`}>
```
Comment (Quiddlee) [inline src/widgets/mobile-view/ui/mobile-view.tsx:51]:
```suggestion
      <Link to=`/${ROUTES.COMMUNITY}/#merch` className={classNames('main-link', color)}>
```

File: src/widgets/mobile-view/ui/mobile-view.tsx [hunk 9]
```diff
@@ -1,6 +1,5 @@
 // todo it not widget
```
Comment (Quiddlee) [inline src/widgets/mobile-view/ui/mobile-view.tsx:1]: Remove this comment, please

File: src/widgets/mobile-view/ui/mobile-view.tsx
Comment (Quiddlee) [inline src/widgets/mobile-view/ui/mobile-view.tsx:1]: Do we also try to refactor `scss` to `modules`?

File: src/widgets/mobile-view/ui/mobile-view.tsx (hunk 9 above)
Comment (natanchik) [reply src/widgets/mobile-view/ui/mobile-view.tsx:1]: Done

File: src/app/app.tsx [hunk 10]
```diff
@@ -7,6 +7,10 @@ import './app.scss';
 const router = createBrowserRouter(routes);
 
 function App() {
+  if (window.location.hostname === 'rollingscopes.com') {
+    window.location.href = 'https://rs.school';
+  }
+
```
[the remark this answers is not in the document]
Comment (natanchik) [reply src/app/app.tsx:13]: Added

File: src/app/layouts/base-layout/components/footer/about-list.tsx [hunk 11]
```diff
@@ -1,30 +1,29 @@
 import { Link } from 'react-router-dom';
-import { buildUrl } from '@/app/services/platform';
 import { LogoWrapper } from '@/shared/ui/logo-wrapper';
 
 const aboutList = [
   {
     title: 'About RS',
-    to: buildUrl('/#about'),
+    to: '/community/#about',
```
[the remark this answers is not in the document]
Comment (natanchik) [reply src/app/layouts/base-layout/components/footer/about-list.tsx:7]: Done

File: src/app/layouts/base-layout/components/footer/about-list.tsx [hunk 12]
```diff
@@ -1,30 +1,29 @@
...
-    to: buildUrl('/#merch'),
+    to: '/community/#merch',
   },
 ];
 
 export const AboutList = () => {
   return (
     <div className="about-list">
-      <Link to={buildUrl('/')}>
+      <Link to="/">
```
[the remark this answers is not in the document]
Comment (natanchik) [reply src/app/layouts/base-layout/components/footer/about-list.tsx:26]: Done

File: src/app/layouts/base-layout/components/header/header.tsx [hunk 13]
```diff
@@ -15,17 +15,17 @@ const cx = classNames.bind(styles);
 const navLinks = [
   {
     label: 'RS School',
-    href: '/#main',
+    href: '/',
```
[the remark this answers is not in the document]
Comment (natanchik) [reply src/app/layouts/base-layout/components/header/header.tsx:18]: Done

File: src/app/layouts/base-layout/components/header/header.tsx [hunk 14]
```diff
@@ -15,17 +15,17 @@ const cx = classNames.bind(styles);
...
     dropdownInner: <SchoolMenu heading="rs school" color="dark" hasTitle={false} />,
   },
   {
     label: 'Courses',
-    href: '/courses/#main',
+    href: '/courses/',
```
[the remark this answers is not in the document]
Comment (natanchik) [reply src/app/layouts/base-layout/components/header/header.tsx:23]: Done

File: src/app/layouts/base-layout/components/header/header.tsx [hunk 15]
```diff
@@ -15,17 +15,17 @@ const cx = classNames.bind(styles);
...
     dropdownInner: <SchoolMenu heading="all courses" color="dark" hasTitle={false} />,
   },
   {
     label: 'Community',
-    href: '/community/#main',
+    href: '/community/',
```
[the remark this answers is not in the document]
Comment (natanchik) [reply src/app/layouts/base-layout/components/header/header.tsx:28]: Done

File: src/widgets/breadcrumbs/ui/breadcrumbs.tsx (hunk 3 above)
Comment (natanchik) [reply src/widgets/breadcrumbs/ui/breadcrumbs.tsx:19]: Done

File: src/widgets/mobile-view/ui/mobile-view.tsx (hunk 7 above)
Comment (natanchik) [reply src/widgets/mobile-view/ui/mobile-view.tsx:45]: Done

File: src/widgets/mobile-view/ui/mobile-view.tsx (hunk 8 above)
Comment (natanchik) [reply src/widgets/mobile-view/ui/mobile-view.tsx:51]: Done

File: src/widgets/mobile-view/ui/mobile-view.tsx (hunk 6 above)
Comment (natanchik) [reply src/widgets/mobile-view/ui/mobile-view.tsx:39]: Done

File: src/widgets/mobile-view/ui/mobile-view.tsx (hunk 5 above)
Comment (natanchik) [reply src/widgets/mobile-view/ui/mobile-view.tsx:25]: Done

File: src/widgets/mobile-view/ui/mobile-view.tsx (hunk 4 above)
Comment (natanchik) [reply src/widgets/mobile-view/ui/mobile-view.tsx:21]: Done

File: src/widgets/mobile-view/ui/mobile-view.tsx
Comment (natanchik) [reply src/widgets/mobile-view/ui/mobile-view.tsx:1]: Let's do it in another task with refactoring

**Review by Yelunina — APPROVED** [review]

**Review by SpaNb4 — APPROVED** [review]

**Pull request merged**